src/views/AddItem.jsx

```diff
@@ -1,3 +1,101 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
```

Comment:
Great work getting the function that submits to firebase and using it in the handleSubmit